src/tests/resistance/random_bytes.rs

```diff
@@ -0,0 +1,86 @@
+use tempfile::TempDir;
+use tokio::time::{sleep, Duration};
+
+use crate::{
+    protocol::codecs::payload::Payload,
+    setup::node::Node,
+    tools::{
+        constants::{
+            ERR_NODE_ADDR, ERR_NODE_BUILD, ERR_NODE_CONNECT, ERR_NODE_STOP, ERR_SYNTH_BUILD,
+            ERR_SYNTH_UNICAST, ERR_TEMPDIR_NEW,
+        },
+        synthetic_node::SyntheticNodeBuilder,
+        util::gen_rand_bytes,
+    },
+};
+
+/// Send some randomly generated to the node before the handshake and check the connection status.
+async fn send_random_data_to_the_node_pre_handshake(len: usize, debug: bool) -> bool {
+    // Spin up a node instance.
+    let target = TempDir::new().expect(ERR_TEMPDIR_NEW);
+    let mut node = Node::builder()
+        .log_to_stdout(debug)
+        .build(target.path())
+        .expect(ERR_NODE_BUILD);
+    node.start().await;
+
+    // Create a synthetic node and disable handshaking.
+    let synthetic_node = SyntheticNodeBuilder::default()
+        .with_handshake(false)
+        .build()
+        .await
+        .expect(ERR_SYNTH_BUILD);
+
+    let net_addr = node.net_addr().expect(ERR_NODE_ADDR);
+
+    // Create a connection without the handshake.
+    synthetic_node
+        .connect(net_addr)
+        .await
+        .expect(ERR_NODE_CONNECT);
+
+    // Send some random data.
+    let random_data_msg = Payload::RawBytes(gen_rand_bytes(len));
+    synthetic_node
+        .unicast(net_addr, random_data_msg)
+        .expect(ERR_SYNTH_UNICAST);
+
+    // Give some time to the node to kill our connection.
+    sleep(Duration::from_millis(1000)).await;
```

Comment:
Maybe this `1000` should be a constant? It's not only for our testing purposes - it has to do with expectation what the node will do.

Reply:
yeah, but this value is tailored just for this test, it can be a local const
Done